From d47e490a68c9faba53b26c379df50211dea57107 Mon Sep 17 00:00:00 2001
From: Roman Shaposhnik <rvs@cloudera.com>
Date: Fri, 30 Mar 2012 09:48:08 -0700
Subject: [PATCH 076/795] MR1: CLOUDERA-BUILD. add pipes (and other c++ libs) to the cdh4 build (CDH-5075)
 (cherry picked from commit 5081a645c0f9572d645e6a8a241508d6ae6bfcad)

(cherry picked from commit 8151709f55d974de5a5f4f421551e17ea112324b)
---
 build.xml |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/build.xml b/build.xml
index a8a1d87..a7fd2eb 100644
--- a/build.xml
+++ b/build.xml
@@ -1384,6 +1384,10 @@
     <copy todir="${dist.dir}/cloudera">
       <fileset dir="cloudera" />
     </copy>
+
+    <copy todir="${dist.dir}/c++" includeEmptyDirs="false">
+      <fileset dir="${build.dir}/c++-examples"/>
+    </copy>
     <!-- END CLOUDERA HOOKS -->
 
     <copy todir="${dist.dir}/.eclipse.templates">
-- 
1.7.0.4

